src/libs/awOrgGroup.js

```diff
@@ -0,0 +1,128 @@
+//
+// mdms-nodejs-client
+//
+// Copyright © 2018 Province of British Columbia
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// Created by Shelly Han on 2018-12-18.
+//
+
+'use strict';
+
+import request from 'request-promise-native';
+import url from 'url';
+import { AW_SUB_URL } from '../constants';
+import { defaultRequestParams } from './utils';
+
+/**
+ * Generate array of child organization groups
+ * @param {Object} orgGroupList the list of org groups in JSON format
+ * @param {Integer} parentOrgID the parent Organization Group ID
+ * @return array of organization groups
+ */
+export const childrenList = (orgGroupList, parentOrgID) => {
+  try {
+    return orgGroupList
+      .filter(org => org.ParentLocationGroup.Id.Value === parentOrgID)
+      .map(org => ({
+        name: org.Name,
+        groupId: org.GroupId,
+        id: org.Id.Value,
+        parentId: org.ParentLocationGroup.Id.Value,
+        uuid: org.Uuid,
+      }));
+  } catch (err) {
+    return [];
+  }
+};
+
+/**
+ * Fetch all organization groups
+ * @param {Object} credential the authentication options
+ * @param {Integer} rootOrgID the root Organization Group ID
+ * @param {Boolean} includeChildren include the array of child org groups
+ * @return array of organization groups
+ */
+export const getAllOrgGroups = async (credential, rootOrgID, includeChildren) => {
```

Comment:
How about we make include children as a default param? `includeChildren = false`?